cookiecutter.json

```diff
@@ -46,8 +42,12 @@
   ],
   "use_async": "n",
   "use_drf": "n",
-  "custom_bootstrap_compilation": "n",
-  "use_compressor": "n",
+  "frontend_pipeline": [
+    "None",
+    "Django Compressor",
+    "Gulp",
+    "Gulp with custom Bootstrap"
```

Comment:
I'm very tempted to merge these 2 and make the custom Bootstrap the default. The Gulp setup without the Bootstrap compilation doesn't provide a lot of value IMO.